packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -91,6 +99,22 @@ const CFDsListing = observer(() => {
     const { has_svg_accounts_to_migrate } = useMT5SVGEligibleToMigrate();
     const getAuthStatus = (status_list: boolean[]) => status_list.some(status => status);
 
+    const getTradingPlatformStatus = async (platform?: string) => {
+        const { trading_platform_status: tradingPlatformStatus } = await WS.send({ trading_platform_status: 1 });
+        const status = tradingPlatformStatus.find((p: { platform: string }) => p.platform === platform)?.status;
```

Comment:
let's use something more descriptive instead of `p`, ambiguity of the code something that needs to be avoided.